rules/linux/defense_evasion_c89_binary.toml

```
@@ -0,0 +1,50 @@
+[metadata]
+creation_date = "2022/03/15"
+maturity = "production"
+updated_date = "2022/03/15"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies Linux binary c89/c99 abuse to break out from restricted environments by spawning an interactive system
+shell.This activity is not standard use with this binary for a user or system administrator. It indicates a potentially
+malicious actor attempting to improve the capabilities or stability of their access
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Linux Restricted Shell Breakout via c89/c99 Shell evasion"
+references = ["https://gtfobins.github.io/gtfobins/c89/","https://gtfobins.github.io/gtfobins/c99/"]
+risk_score = 47
+rule_id = "1859ce38-6a50-422b-a5e8-636e231ea0cd"
+severity = "medium"
+tags = ["Elastic", "Host", "Linux", "Threat Detection", "Defense Evasion", "GTFOBins"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+sequence by host.id, process.pid with maxspan=1m
+[process where process.name :("c89","c99") and process.args == "-wrapper" and process.args : ("sh,-s", "bash,-s", "dash,-s", "/bin/sh,-s", "/bin/bash,-s", "/bin/dash,-s")]
+[process where process.parent.name : ("bash", "sh", "dash")]
```

```suggestion
[process where process.name : ("c89","c99") and process.args : "-wrapper" and
   (process.args : ("sh", "bash", "dash") and
   process.args : "-s")]
[process where process.parent.name : ("bash", "sh", "dash")]
```

We can discuss this one, but looks like `sh,-s` args could be bypassed with spaces, would `sh, -s` work?